app/src/main/java/io/github/hidroh/materialistic/WebFragment.java

```diff
@@ -196,6 +214,10 @@ private void bindContent() {
         }
     }
 
+    private void removeRefreshing() {
```

Comment:
This never gets executed ! I thought ItemResponseListener would be called after the item is loaded.